java/src/test/java/scenario/ScenarioSteps.java

```diff
@@ -149,65 +164,59 @@ public ScenarioSteps() throws IOException {
-                            "--version", version, "--signature-policy", "AND(\"Org1MSP.member\",\"Org2MSP.member\")",
-                            "--sequence", "1", "--waitForEvent");
-                    approveCommand.addAll(tlsOptions);
-                    exec(approveCommand);
-
-                    // Org2
-                    exec("docker", "exec", "org2_cli", "peer", "lifecycle", "chaincode", "package", ccPackage, "--lang",
-                            ccType, "--label", ccLabel, "--path", ccPath);
-
-                    exec("docker", "exec", "org2_cli", "peer", "lifecycle", "chaincode", "install", ccPackage);
-
-                    installed = exec("docker", "exec", "org2_cli", "peer", "lifecycle", "chaincode", "queryinstalled");
-                    matcher = regex.matcher(installed);
-                    if (!matcher.matches()) {
-                        System.err.println(installed);
-                        throw new IllegalStateException("Cannot find installed chaincode for Org2: " + ccLabel);
-                    }
-                    packageId = matcher.group(1);
-
-                    approveCommand = new ArrayList<>();
-                    Collections.addAll(approveCommand, "docker", "exec", "org2_cli", "peer", "lifecycle", "chaincode",
-                            "approveformyorg", "--package-id", packageId, "--channelID", channelName, "--name", ccName,
-                            "--version", version, "--signature-policy", "AND(\"Org1MSP.member\",\"Org2MSP.member\")",
-                            "--sequence", "1", "--waitForEvent");
-                    approveCommand.addAll(tlsOptions);
-                    exec(approveCommand);
 
                     // commit
                     List<String> commitCommand = new ArrayList<>();
-                    Collections.addAll(commitCommand, "docker", "exec", "org1_cli", "peer", "lifecycle", "chaincode",
-                            "commit", "--channelID", channelName, "--name", ccName, "--version", version,
-                            "--signature-policy", "AND(\"Org1MSP.member\",\"Org2MSP.member\")", "--sequence", "1",
-                            "--waitForEvent", "--peerAddresses", "peer0.org1.example.com:7051", "--peerAddresses",
-                            "peer0.org2.example.com:8051", "--tlsRootCertFiles",
+                    Collections.addAll(commitCommand, "docker", "exec", "org1_cli", "peer", "lifecycle", "chaincode", "commit",
+                            "--channelID", channelName,
+                            "--name", ccName,
+                            "--version", version,
+                            "--signature-policy", signaturePolicy,
+                            "--sequence", "1",
+                            "--waitForEvent",
+                            "--peerAddresses", "peer0.org1.example.com:7051",
+                            "--peerAddresses",
```

Comment:
Do we also need org3 here now we're a 3 org network?

Reply:
We could add it, but don't need to.  It just needs a majority of endorsers